app/src/main/java/com/naxtlevelofandroiddevelopment/marvelgallery/presentation/common/PresenterBaseActivity.kt

```diff
@@ -0,0 +1,24 @@
+package com.naxtlevelofandroiddevelopment.marvelgallery.presentation.common
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+
+abstract class PresenterBaseActivity : AppCompatActivity() {
```

Comment:
i would remove this base class, callbacks should be only called when needed by given presenter